apps/antalmanac/src/components/RightPane/CoursePane/CoursePaneButtonRow.tsx

```diff
@@ -65,20 +66,24 @@ export function ColumnToggleButton() {
         store.selectedColumns,
         store.setSelectedColumns,
     ]);
-    const [open, setOpen] = useState(false);
+    const [anchorEl, setAnchorEl] = useState<HTMLElement>();
+    const open = useMemo(() => Boolean(anchorEl), [anchorEl]);
 
-    const handleChange = useCallback((e: SelectChangeEvent<SectionTableColumn[]>) => {
-        if (typeof e.target.value !== 'string') {
-            setSelectedColumns(e.target.value);
-        }
-    }, []);
+    const handleChange = useCallback(
+        (e: SelectChangeEvent<SectionTableColumn[]>) => {
```

Comment:
If you check the type signature for MUI's select `onChange` prop, the callback function should receive the value itself as the second argument. It's easier to work with the value itself if that's all you need, rather than extract it from the event.

```suggestion
        (e: unknown, value: SectionTableColumn[]) => {
```

Reply:
😭 i can't make this work so feel free to take a crack at it; I'm getting spit nasty errors (and also it just doesn't work idk)